pkg/client/public/locales/en/translation.json

```diff
@@ -107,6 +107,7 @@
         "appNotAssesedTitle": "Assessment has not been completed",
         "appNotAssessedBody": "In order to review an application it must be assessed first. Please assess the application",
         "assessmentStakeholderHeader": "Select the stakeholder(s) or stakeholder group(s) associated with this assessment.",
+        "binaryPackaging": "If undefined Packaging will default to 'jar'.",
```

Comment:
A minor nit here - I think this should read: 

"Packaging will default to JAR if left empty." 

or something like that.